### DISCUSSION TITLE
Anything Need Done?

### DISCUSSION THREAD

**Discussion by noraa-july-stoke** [wilfredinni/python-cheatsheet#168] [General]
Is there any agenda or task list to look at for this repo?

**Comment by wilfredinni** [comment]
Hey! I just published the [Roadmap](https://github.com/users/wilfredinni/projects/2).

**Reply by noraa-july-stoke** [reply]
awesome! I am going to take a look at it. by the way do you have a contact email or something i can direct questions to? i was just wondering cause ive sent a few emails to the carlos@pythoncheatsheet email but have never gotten a response so i didnt know if you were just busy or you dont check that email.

**Reply by wilfredinni** [reply]
Hi! I may be a bit slow on responding because of my job, but I will.

**Reply by noraa-july-stoke** [reply]
I definitely understand and no worries, I just wanted to make sure it was still valid! I know this is probably just something you maintain in your spare time. It's an amazing resource for people and I'm happy to help any way I can 🙃
